app/uk/gov/hmrc/nationaldutyrepaymentcenter/models/FileTransferRequest.scala

```diff
@@ -29,10 +29,7 @@ case class FileTransferRequest(
                                               fileMimeType: String,
                                               batchSize: Int,
                                               batchCount: Int,
-                                              correlationId: Option[String] = None,
-                                              // private field, this value will be overwritten
-                                              // with x-request-id header value in the controller
-                                              requestId: Option[String] = None
```

Comment:
Presume the `requestId` wasn't used here?

Reply:
Correct

This class was copy/pasted from the File Transmission Synchronous service, which populates the `requestId` (a "private field") based on the incoming `x-request-id` header (overwriting whatever was de-serialised from the incoming payload)

https://github.com/hmrc/file-transmission-synchronous/blob/master/app/uk/gov/hmrc/traderservices/controllers/FileTransferController.scala#L64